README.md

```diff
@@ -62,38 +62,76 @@ forge create --rpc-url ${NODE_RPC_URL} --private-key ${DEPLOYER_PRIVATE_KEY} --u
 [⠒] Compiling...
-[⠢] Compiling 4 files with 0.8.23
-[⠆] Solc 0.8.23 finished in 2.04s
+No files changed, compilation skipped
 Compiler run successful!
 Deployer: 0x982A41a1F3bC1F8bdB71F11751F3a71691794AfA
-Deployed to: 0x4F0e62B7294D26b223a2cffc02BE5D072528c0De
-Transaction hash: 0xcf3c71f9b71fff6f7f45dbc3a512db667e10ab6eb272a700dfe1ab32aef0958c
+Deployed to: 0x4d21849008fa59F36971A2611746F080b9B79220
+Transaction hash: 0x7a6d29b281aaef31e5d7ae5080b8800614f4f053b63fbb83a0d870807197b252
 ```
 
 ### Verify PasskeyAccountFactory contract
 
-- Replace `<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>` with your deployed PasskeyAccountFactory address.
+- Replace `<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>` with your deployed PasskeyAccountFactory address (e.g., 0x4d21849008fa59F36971A2611746F080b9B79220).
 
 ```shell
-forge verify-contract --watch --chain "sepolia" --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version "0.8.23" --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) "<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>" "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
+PASSKEY_ACCOUNT_FACTORY_ADDRESS="<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>"
+
+forge verify-contract --watch --chain "sepolia" --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version "0.8.23" --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) ${PASSKEY_ACCOUNT_FACTORY_ADDRESS} "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
 ```
 
 - Output sample
 
 ```shell
-Start verifying contract `0x4F0e62B7294D26b223a2cffc02BE5D072528c0De` deployed on sepolia
+Start verifying contract `0x4d21849008fa59F36971A2611746F080b9B79220` deployed on sepolia
 
-Submitting verification for [src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory] 0x4F0e62B7294D26b223a2cffc02BE5D072528c0De.
+Submitting verification for [src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory] 0x4d21849008fa59F36971A2611746F080b9B79220.
 Submitted contract for verification:
         Response: `OK`
-        GUID: `u27yc7avgdwdce827ptgvhwtbat1daqzrwh362xp2zganicbrw`
-        URL:
-        https://sepolia.etherscan.io/address/0x4f0e62b7294d26b223a2cffc02be5d072528c0de
-Contract verification status:
-Response: `NOTOK`
-Details: `Pending in queue`
+        GUID: `tqw8mnuhqwuigkkp9vwzgu89qespbkma6umeszbmjlvcjmvj1i`
+        URL: https://sepolia.etherscan.io/address/0x4d21849008fa59f36971a2611746f080b9b79220
 Contract verification status:
 Response: `OK`
 Details: `Pass - Verified`
 Contract successfully verified
 ```
+
+### Deploy PasskeyAccount contract via PasskeyAccountFactory
+
+```shell
+cast send --rpc-url ${NODE_RPC_URL} --private-key ${DEPLOYER_PRIVATE_KEY} ${PASSKEY_ACCOUNT_FACTORY_ADDRESS} "createAccount(string calldata credId,uint256 pubKeyX,uint256 pubKeyY,uint256 salt)" ${PASSKEY_CREDENTIAL_ID} ${PASSKEY_X} ${PASSKEY_Y} 0
+```
+
+### Verify PasskeyAccount implementation contract
```

Comment:
This only needs to do once for each factory, right? What about putting it after factory verification?